### DISCUSSION TITLE
Update latest tag

### DISCUSSION THREAD

**Discussion by Shizanit** [mastodon/mastodon#26969] [General]
![image](https://github.com/mastodon/mastodon/assets/65107731/eeaf9edf-3cd7-45c9-ae3d-106108825883)

**Comment by ClearlyClaire** [comment]
Where are you seeing this?

**Reply by Shizanit** [reply]
https://github.com/mastodon/mastodon/pkgs/container/mastodon

**Reply by ClearlyClaire** [reply]
what i'm seeing is:
![image](https://github.com/mastodon/mastodon/assets/384364/6d849a93-8f79-495d-a820-6bdd3bfc309a)

**Reply by Shizanit** [reply]
and my server also sees the old image on docker pull

**Reply by renchap** [reply]
What are you seeing on this page: https://github.com/mastodon/mastodon/pkgs/container/mastodon/125089263?tag=latest ?

**Reply by Shizanit** [reply]
oops sorry confused repositories with https://github.com/glitch-soc/mastodon , github interface update is very confusing

**Reply by ClearlyClaire** [reply]
Alright, there is no issue with this repo, but with glitch-soc which does not have proper releases and thus does not update the `latest` tag. I will fix it on glitch-soc's side.
